docusaurus.config.js

```diff
@@ -64,19 +64,19 @@ const config = {
                     position: "left",
                 },
                 {
-                    href: 'https://docs.talawa.io/talawa/index.html',
-                    target: "_self",
+                    to: 'https://docs.talawa.io/talawa/index.html',
+                    target: "_blank",
                     label: "Talawa",
                     position: "left",
                 },
                 {
-                    href: 'https://docs.talawa.io/talawa-api/schema/index.html',
```

Comment:
Remove `https://docs.talawa.io` to make the link relative. That way it won't break the site if we move it.